.github/workflows/macos-arm.yml

```diff
@@ -52,6 +52,9 @@ jobs:
                 python3 -m pip install ninja
                 python3 -m pip install --upgrade --pre numpy
 
+            - name: Get specific version of CMake, Ninja
```

Comment:
We are only getting CMake here.

Reply:
Not Ninja.

Reply:
No, we do get Ninja.

Reply:
We get ninja a few lines above, and not a specific version either.

Reply:
We do not need to get it above.